.rubocop.yml

```diff
@@ -30,6 +30,9 @@ Metrics/BlockLength:
     - "config/**/*"
     - "spec/**/*"
 
+Metrics/ParameterLists:
+  Enabled: false
```

Comment:
why did you disable this? Revert this change.

Reply:
Got it. Better to pass a hash parameter for this one instead of passing 9 parameters. That is a bad practice.